src/api/controllers/employers.js

```diff
@@ -22,7 +18,46 @@ function getEmployerByAuthUser(req, res) {
                 res.status(200).json(employer);
             });
         });
-
+    } else if (req.user.user_type === 'student') {
```

Comment:
What's the use case for this? why don't we just have a GET /users/auth/favorites route? If its 1:n it makes more semantic sense to have it on the user controller

```
// GET /users/auth/favorites
function getFavoritesForAuthUser() {
    User.findById(req.user._id).populate('favorites').exec(function(err, user){
        return res.status(200).json({
            "favorites": user.favorites
        })
    })
}
```

Reply:
The reasoning was that it would be convenient for the client to have this info included right away with the employer. 

Thinking about it now though, it makes more sense to just let the client use the user's favorites list to figure this out individually for each employer. Especially in the context of actually toggling favorite status or not per company.